.github/workflows/ci.yml

```diff
@@ -25,6 +26,6 @@ jobs:
           node-version: 12
 
       - name: Run ${{ matrix.command }}
-        run: yarn && yarn build:rollup && yarn ${{ matrix.command }}
+        run: yarn && yarn build && yarn ${{ matrix.command }}
         env:
           CI: true
```

Comment:
Bring `env: CI: true` up to the root of the `ci.yml` please. Makes this step a tiny bit cleaner.